dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-k8s.ts

```diff
@@ -59,6 +59,18 @@ export function useK8s(model: { [field: string]: any }): IJsonItem[] {
         message: t('project.node.min_memory_tips')
       }
     },
+    {
+      type: 'input',
+      field: 'command',
+      name: t('project.node.command'),
+      props: {
+        placeholder: t('project.node.command_tips')
+      },
+      validate: {
+        trigger: ['input', 'blur'],
+        message: t('project.node.command_tips')
+      }
```

Comment:
what is this to validate? required?

Reply:
Hi, @Amy0104 Thanks for the comment.

Actually, this validate is copied from the above code. (I am not familiar with front-end code)

<img width="661" alt="截屏2022-10-31 14 29 12" src="https://user-images.githubusercontent.com/38122586/198945629-ba34aee8-dbf7-4b93-bb7d-03bdf787ff3f.png">


Because this field `command` is not necessary, user can leave it blank.

So we just remove this validate code as below, right?


<img width="477" alt="image" src="https://user-images.githubusercontent.com/38122586/198946282-6bba2aa6-d9bb-4770-994f-cf5b04f504dc.png">

Reply:
It's better to remove it, if u don't need it.

Reply:
> It's better to remove it, if u don't need it.

ok, i got it. Thanks again for your comment~